### ISSUE TITLE
JSON file with MCParticle tree cannot be opened

### ISSUE THREAD

**Issue by tmadlener** [key4hep/eede#34]
This is not the highest priority, but we have encountered an issue with opening some json files, e.g. the following one (which contains a rather large MCParticle tree from a calorimeter shower). I don't think the size of the file is a problem, but I suspect that potentially a slightly newer version of podio or EDM4hep could be one?

[test_mc_shower.edm4hep.json](https://github.com/user-attachments/files/15585039/test_mc_shower.edm4hep.json)

What we get is the following error:
![image](https://github.com/key4hep/dmx/assets/16774861/3ddcb223-83b4-4a84-8d3e-7172810989fa)

If you know where the error is coming from, I can also try to have a look myself to see if I can fix this.

**Comment by brauliorivas** [comment]
So it turns out that the file only has `Event 1`, so when selecting an event, it automatically chooses 0, but it doesn't exist 😄. However, in that case we could improve the event input, and automatically select the first event it finds.

**Comment by tmadlener** [comment]
Ah right, I thought I had tried to go to `1` in the selection with the same issue, but it does indeed seem to work if I select the event number. Can the file be partially read to populate a dropdown menu to select the event?

Another issue that I observed with this JSON file is that at least chrome really struggles to display it (goes up to 3GB of RAM for me and I still don't see a graph :sweat_smile:). Might just be that this file with approx. 18k MCParticles is too big to be displayed. I wouldn't too much emphasis on the performance part yet, but at least we would have an input file to use for performance testing and developments.
